src/_posts/2019-06-12-progress-structure.md

```diff
@@ -0,0 +1,52 @@
+
+One of the things that we decided at the beginning is that we were not trying to create the perfect structure. We wanted to create something that was good enough, and we will keep evaluating the system and improving it once we started to see the issues with it. 
+
+The other important aspect is that we are a consultancy, and therefore the skills chosen for advancement are to improve our capacity as consultants.
+
+## Progress Structure.
+So how is the progress structure setup? We have a collection of disciplines—areas where we want people to improve. Each discipline is divided into at most 5 levels. Each level has one or more skills/experiences. The idea is that they represent a continuous advancement on the difficulty or knowledge required. Then there are the spider nets, an approximate shape of the levels on each discipline that we expect a person at each level to have reached. We want to allow for some flexibility as some of the “requirements” really depend on being provided with opportunities to develop in these areas. Sometimes people are so well advanced around a specific area that compensates for lack in others. But in general, for a consultancy, well rounded individuals, that can cover all the job/projects possibilities is better.
+
+A detail that I believe is important is that your career advancement will not depend on someone of higher seniority level vacating a position. Each person progress is its own, and will not be affected by others progression.
+
+Originally I saw this as trying to get well rounded people that is useful on every possible scenario. But probably the broken comb metaphor is better suited on the long term. The less people you have in your company, the more important it is that anyone can do any job. But as you grow, that importance subsides.
+
+Finally, the competency matrix that we are using is not a tick-all-boxes idea. It provides a general idea of  expectations.
+
+##  It is a Marathon, not a sprint.
+I think this is something that needs to be understood. It takes years to reach Principal (the highest seniority) level. You need a broad knowledge and experience. You need to have seen multiple projects, with different characteristics, so for any new assignments you can reach into your own drawer of solutions, approaches, tricks to move forward satisfactorily a project. Trying to race here only leads to disappointment. Or as they saying goes: Rome wasn’t built in a day.
+
+## Mentorship
+Mentoring is one of the most important ways in which we can help people to improve and progress. Therefore, we are introducing mentoring (non-compulsory) for every level of seniority. A mentor will become a guide and helper, to allow the mentee to move forwards. It is important however that the mentee the one in charge of his/her own progression. The mentor is there to lend a hand, provide advice, and help with getting Codurance to provide the resource/opportunities/solutions that the mentee required.
```

Comment:
"that the mentee required." -> "that the mentee requires."

Reply:
done